src/components/screens/lists/Lists.tsx

```diff
@@ -1,52 +1,52 @@
 "use client";
 
-import { type NextPage } from "next";
-import { FC, PropsWithChildren, useState } from "react";
+import { FC, ReactElement, useEffect } from "react";
 import FilterListIcon from "@mui/icons-material/FilterList";
 import FileDownloadOutlinedIcon from "@mui/icons-material/FileDownloadOutlined";
 import classNames from "classnames/bind";
+import { useRouter } from "next/navigation";
 
 import styles from "./lists.module.scss";
 
-import SelectedCustom, {
-  TSelectedItem,
-} from "@/components/reused/selected-custom/SelectedCustom";
-import { listOption } from "@/components/screens/lists/listOption";
-
-import {
-  IconSmallRightArrow,
-  IconTriangleArrow,
-} from "@/components/reused/Icon/Icon";
-
 import { SearchField } from "@/components/reused/fields/search/SearchField";
-import Link from "next/link";
 import { TeacherList } from "@/components/ui/lists/teacher-list/TeacherList";
-import { usePathname } from "next/navigation";
 import { StudentList } from "@/components/ui/lists/student-list/StudentList";
 import { GroupList } from "@/components/ui/lists/group-list/GroupList";
+import { listOption } from "@/components/screens/lists/listOption";
+import { listTypeEnum } from "@/enums/lists/listType-enum";
+import { outputError } from "@/services/output-error";
+import { getToastify } from "@/services/toastify";
+import { ToastifyEnum } from "@/enums/toastify-enum";
 
 const cx = classNames.bind(styles);
 
 type Props = {
   type: string | null;
 };
 
+const listTabs: { [key: string]: ReactElement } = {
```

Comment:
const listTabs: { [key in listTypeEnum]: ReactElement } = {
  [listTypeEnum.TEACHER]: <TeacherList />,
  [listTypeEnum.STUDENT]: <StudentList />,
  [listTypeEnum.GROUP]: <GroupList />,
};